README.md

```diff
@@ -343,15 +345,14 @@ Sometimes `TSK Extractor` can't talk to the car. Try again.
 
 ### Step 5. Install `commaai/nightly-dev`
 
-> [!WARNING]
-> **Wherever `commaai/nightly-dev` is seen, C3 users should instead use Sunnypilot's `sunnypilot/staging-c3-new` wherever they see `commaai/nightly-dev`. comma dropped support for the C3 in their codebase. [This guide will be getting some updates to reflect this change and an issue has been made to track that.](https://github.com/optskug/docs/issues/51). C3X users should continue with `commaai/nightly-dev`.**
-
 Start your car's engine.
 
 Go to the `Reboot Menu` and `Install commaai/nightly-dev`.
 
 `commaai/nightly-dev` is the only branch from comma.ai with TSK support.
 
+If you have C3, install `sunnypilot/staging-tici`. There are no branches from comma.ai with C3+TSK support, so SunnyPilot is your best option. If you have no idea what this means, come talk to us in [comma Discord](https://discord.comma.ai) at #toyota-security channel.
```

Comment:
I suspect most C3 owners would be experienced users who's just moving the C3 to their secondary car

Reply:
I think its fine. We can edit the guide by blood.